component/core/src/main/java/org/exoplatform/social/core/space/SpaceException.java

```diff
@@ -76,7 +76,10 @@
 
     /** The SPAC e_ alread y_ exist. */
     SPACE_ALREADY_EXIST,
-    
+
+    /** INVALID space name length */
+    INVALID_SPACE_NAME_LENGTH,
```

Comment:
Let's make the error more generic
```suggestion
    INVALID_SPACE_NAME,
```